packages/website/src/components/ast/PropertyName.tsx

```diff
@@ -9,21 +9,23 @@ export interface PropertyNameProps {
 }
 
 export default function PropertyName(props: PropertyNameProps): JSX.Element {
+  const { onClick: onClickProps, onHover } = props;
```

Comment:
Destructuring props to avoid stale values as suggested by:  https://github.com/facebook/react/issues/16265#issuecomment-517518539